app/ide-desktop/client/src/server.ts

```diff
@@ -523,7 +514,7 @@ export class Server {
       const parentDirectory = path.join(projectsDirectory, `cloud-${projectId}`)
       await access(parentDirectory)
         .then(() => {
-          rmdir(parentDirectory, { maxRetries: 3, recursive: true })
+          rm(parentDirectory, { maxRetries: 3, recursive: true, force: true })
```

Comment:
`recursive` parameter of `rmdir` function is [deprecated](https://nodejs.org/api/fs.html#fspromisesrmdirpath-options)